### ISSUE TITLE
How to create flow.txt and rgb.txt files?

### ISSUE THREAD

**Issue by Gavin666Github** [USTC-Video-Understanding/I3D_Finetune#17]
I've extracted the images of RGB and flow to the local path, but how do I create the corresponding TXT file? anyone who can share the  script?Thank you very much!

**Comment by chunduri11** [comment]
@Gavin666Github 
How did you extract RGB and FLOW  images?

I have written a code using opencv to do it, but takes long time to generate flow images. And also the the size of flow images are very big.

I am also looking for an answer to how the .txt files can be generated with a script.
thanks

**Comment by DSOCT18** [comment]
can you tell me how to get the flow data and from where?I am trying to use my own custom dataset.
can we just run the rgb finetune and get good results? or is flow necessary?

**Comment by rrryan2016** [comment]
@Gavin666Github  @chunduri11 
heyyyy there! Could you please share me your experience in how to extract RGB and flow? great thx

**Comment by DSOCT18** [comment]
> @Gavin666Github @chunduri11
> heyyyy there! Could you please share me your experience in how to extract RGB and flow? great thx

https://github.com/qijiezhao/py-denseflow.git
this might help you..

**Comment by rrryan2016** [comment]
> 
> 
> > @Gavin666Github @chunduri11
> > heyyyy there! Could you please share me your experience in how to extract RGB and flow? great thx
> 
> https://github.com/qijiezhao/py-denseflow.git
> this might help you..

Cheers man. 
I am doing this, I notice but not quite sure about that will it generate both RGB and flow data ?

**Comment by DSOCT18** [comment]
> > > @Gavin666Github @chunduri11
> > > heyyyy there! Could you please share me your experience in how to extract RGB and flow? great thx
> > 
> > 
> > https://github.com/qijiezhao/py-denseflow.git
> > this might help you..
> 
> Cheers man.
> I am doing this, I notice but not quite sure about that will it generate both RGB and flow data ?

yes it will generate both rgb and flow images in one folder..you will have to manipulate paths to get rgb and flow in different folders.

**Comment by sanolans** [comment]
@Gavin666Github @rrryan2016 @chunduri11 @DSOCT18 @vra Hi guys! can you guys tell me how to generate the image list from the directory? Thanks.

**Comment by sanolans** [comment]
https://github.com/USTC-Video-Understanding/I3D_Finetune/issues/31#issuecomment-567837549

Check the link to see the code for generating rgb.txt and flow.txt
